api-reference/10 UI Widgets/dxFileUploader/1 Configuration/dialogTrigger.md

```diff
@@ -5,7 +5,38 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the HTML element a click on which invokes file upload dialog.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+You can use any of the following ways to specify the **dialogTrigger** option:
```

Comment:
What do you mean by "ways"? What are these 'ways'?

Reply:
ways to specify the option value: as a string, jQuery object etc.

Reply:
Then you should say something like "You can use a string, jQuery object, and... to specify the **dialogTrigger** option:"

Reply:
ok